src/templates/landing_pages/binary_in_numbers.html.tt

```diff
@@ -0,0 +1,46 @@
+<div class="gr-padding-10 static_full">
+    <h1 class="center-text">Binary.com in Numbers</h1>
```

Comment:
we don't care about translations?
also, instead of any `Binary.com` please use a placeholder with variable `website_name`